package.json

```diff
@@ -52,36 +52,36 @@
     "uuid": "^8.3.2"
   },
   "peerDependencies": {
-    "jest": "^26.6.3",
-    "jest-circus": "^26.6.3",
-    "jest-environment-node": "^26.6.2",
-    "jest-runner": "^26.6.3"
+    "jest": ">=26.6.3",
+    "jest-circus": ">=26.6.3",
+    "jest-environment-node": ">=26.6.2",
+    "jest-runner": ">=26.6.3"
```

Comment:
Shouldn't these, at least `jest-runner`, really be set to >=27.0.0? If users now use Jest 26 and don't set e.g. `"testRunner": "jest-circus/runner"` themselves, it means many of the things in the environment such as `handleTestEvent` will not work, right?

Reply:
@jhildenbiddle I suppose that it should be work with Jest 26, but only if you set `"testRunner": "jest-circus/runner"` manually. But i need to check this. And put it in documentation. Thanks!